Installing needle dependencies, apt-get (#1091)
OWASP/owasp-mstg: Document/0x06b-Basic-Security-Testing.md

- **SAVE_HISTORY**: If set to "true," the command history will persist across sessions.
- **VERBOSE, DEBUG**: If set to "true," this will enable verbose and debug logging, respectively.

**Troubleshooting**

In order to use the modules in Needle, you may have to install its dependencies. Use this command in Needle:

> ```

Reviewer: change to ```shell

use device/dependency_installer

run
```

> Other modules may promt you the ```apt-get``` command has not been installed. To get ```apt-get```, go to your Cydia and look for  ```CyDelete``` and install it. 

Reviewer: Can you change ``` to ` on line 290?

- **SAVE_HISTORY**: If set to "true," the command history will persist across sessions.
- **VERBOSE, DEBUG**: If set to "true," this will enable verbose and debug logging, respectively.

> **Trouble shooting**

Reviewer 2: ```suggestion
**Troubleshooting**
```

> Please pay attention, in order to use the modules in Needle. You might have to install its dependencies by using this command in Neelde.

Reviewer 2: `Neelde` should be `Needle`.

I would remove "Please pay attention", as it doesn't add much.

I think that the following parts are particularly related: "in order to use the modules in Needle" and "You might have to install its dependencies". I would put them in the same sentence.

```suggestion
In order to use the modules in Needle, you may have to install its dependencies. Use this command in Needle:
```

```
use device/dependency_installer

run
```

> Other module may promt you the ```apt-get``` command has not been installed. To get ```apt-get```, go to your Cydia and look for  ```CyDelete``` and install it. 

Reviewer 2: `promt` should be `prompt`. 

> look for  ```CyDelete``` and install it

Is this correct? CyDelete looks to me like something that deletes things, not something that installs things.

Author: You'r correct. But in order to install ``apt-get``` command, this package ```CyDelete``` will install it for us. Well, it could be other ways to get the ```apt-get``` installed though. For me, installing ```CyDelete``` is the fastest way.

> This will install the needed dependencies, command lines for later use.

Reviewer 2: I am not clear on what this sentence means, especially the part "command lines for later use".

Author: Thanks for pointing it out. I should have not put it there. It makes people confused.

Author: Fixed! I removed the sentence to not make people get confused
